README.md

```diff
@@ -87,5 +87,6 @@ This repository is designed to help newbies practice using Git and GitHub by cre
 ## Contributors
 
 - [John Doe](./john-doe/README.md)
+- [Lucas Marvel](./lucas/README.md)
```

Comment:
you name should be after `Seb` or `White`

Reply:
I fixed it